kubectl-rook-ceph.sh

```diff
@@ -493,6 +495,50 @@ function run_debug() {
     ;;
   esac
 }
+
+function run_dr_clusters_health() {
+  [[ -z "${1:-""}" ]] && fail_error "Missing subcommand"
+  subcommand="$1"
+  shift # remove the subcommand from the front of the arg list
+  case $subcommand in
+  health)
+    run_dr_health "$@"
+    ;;
+  *)
+    fail_error "'dr' subcommand '$subcommand' does not exist"
+    ;;
+  esac
+}
+
+function run_dr_health() {
+  # do not call end_of_command_parsing here because all remaining input is passed directly to 'ceph'
+  info_msg "fetching the cephblockpool with mirroing enable"
+  blockpool_List=$(KUBECTL_NS_CLUSTER get cephblockpool | awk '{print $1}' | sed "1 d")
+  blockpool_Name=""
+  for blockpool in $blockpool_List; do
+    mirroring_Enabled=$(KUBECTL_NS_CLUSTER get cephblockpool "$blockpool" -o jsonpath='{.spec.mirroring.enabled}')
+    if [ "$mirroring_Enabled" == "true" ]; then
+      blockpool_Name+=$blockpool
+      info_msg "found '$blockpool' cephblockpools with mirroing enabled"
+      break
+    fi
+  done
+
+  [ -z "$blockpool_Name" ] && error_msg "DR is not confiqured, cephblockpool with mirroring enabled not found."
```

Comment:
Let's use all lower case for the vars with the underscore
```suggestion
  [ -z "$blockpool_name" ] && error_msg "DR is not confiqured, cephblockpool with mirroring enabled not found."
```